devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -188,9 +188,20 @@ private List<Map> handleAcceleration(descMap) {
 
 private List<Map> parseAxis(List<Map> attrData) {
 	def results = []
-	def x = hexToSignedInt(attrData.find { it.attrInt == 0x0012 }?.value)
-	def y = hexToSignedInt(attrData.find { it.attrInt == 0x0013 }?.value)
-	def z = hexToSignedInt(attrData.find { it.attrInt == 0x0014 }?.value)
+
+	def attrsFound = []
+	for (attrId in [0x0012, 0x0013, 0x0014]) {
```

Comment:
So, using a for loop is not typically the groovy way, and it also makes debugging from sumo traces tougher (since we will only have the line number, and won't know where in the loop it is).  I would instead, leave the definitions of x, y, and z the same and do something like

```
if ([x,y,z].findIndexOf { it == null } != -1) {
    return []
}
```